STAC-24134: Add release notes 2.7.0 release and update with staging changes (#172)
rancher/stackstate-product-docs: docs/latest/modules/en/pages/setup/release-notes/v2.7.0.adoc

** *Customization:* Supports query variables, custom thresholds, and visualization styling. Dashboards can be bundled in StackPacks or created ad-hoc from existing metric widgets.
* *CLI Management:* The `sts` CLI now supports the `dashboard` command to manage SUSE Observability Dashboards programmatically.
* *Quorum Observation Daemon:* Introduced a small logging daemon that monitors the quorum of redundant services within the platform to ensure high availability.
* *Improved Error Handling:* Enhanced error messages and handling across all platform perspectives.
* *Storage Metric Visualization:* Improved PV and PVC metric charts to simplify data by showing only a single time-series for usage and capacity.
* *Agent: Splunk Integration:* Disabled saved searches in Splunk are now automatically ignored by the Splunk check.

== Bug Fixes

* *Platform: SLES 16 OOM Fix:* Resolved an issue where several pods (HBase, Tephra) would encounter Out-Of-Memory (OOM) errors on SLES 16 due to cgroups v2 resource management.
* *Backup Configuration Redesign:*

> ** *BREAKING CHANGE:* The `backup.enabled` Helm value has been replaced by `global.backup.enabled`. Using the old `backup.enabled=true` value will now cause the deployment to **fail**.

Reviewer: This whole section for backup configuration is not a bug fix but a new feature/enhancement

= v2.7.0 - 14/Jan/2026
:revdate: 2026-01-14
:page-revdate: {revdate}
:description: SUSE Observability Self-hosted

== Release Notes: {stackstate-product-name} Helm Chart v2.7.0

WARNING:  This release has a *BREAKING CHANGE*: The `backup.enabled` Helm value has been replaced by `global.backup.enabled`. Using the old `backup.enabled=true` value will now cause the deployment to **fail**.

== Dashboards

SUSE Observability now supports dashboards. A major new feature allowing the visualization of data from business processes, user flows, and data pipelines in a central location. This provides a familiar environment for users migrating from tools like Grafana and Prometheus to recreate visualizations and define custom metrics.

* *Integrated RBAC:* Dashboards respect existing platform permissions and can be toggled between private and public visibility.
* *Versatile Widgets:* Includes time-series charts, bar charts, gauges, and Markdown fields with support for manual resizing and drag-and-drop layout.
* *Customization:* Supports query variables, custom thresholds, and visualization styling. Dashboards can be bundled in StackPacks or created ad-hoc from existing metric widgets.

> For more information on dashboards xref:use/dashboards/dashboards.adoc[dive into the documentation and get started].

Reviewer 2: ```suggestion
For more information on dashboards, see xref:use/dashboards/dashboards.adoc[dashboard documentation].
```

* *Agent: Splunk Integration:* Disabled saved searches in Splunk are now automatically ignored by the Splunk check.

== Bug Fixes

* *Platform: SLES 16 OOM Fix:* Resolved an issue where several pods (HBase, Tephra) would encounter Out-Of-Memory (OOM) errors on SLES 16 due to cgroups v2 resource management.
* *Platform: Kafka Init Container:* Fixed a bug where the Kafka init container would occasionally get stuck during a fresh installation.
* *StackGraph: Long-Running Scans:* Resolved a StackGraph corruption issue that could occur during long-running data scans.
* *Platform: RBAC Agent Token Refresh:* Fixed an issue where the `rbac-agent` failed to refresh its ServiceAccount token, leading to a loss of authentication.
* *UI Performance:* Resolved an issue where the user interface would occasionally become unresponsive when viewing complex metric charts.
* *OpenShift StackPack Fix:* Corrected the StackPack page URL for integration with the OpenShift `ClusterLogForwarder`.
* *Slack Notification Improvement:* Users can now manually enter a Slack Channel ID, resolving issues when workspaces contain a large number of channels.

> * *YAML Variable Parsing:* Fixed a bug in YAML parsing when a value is a variable reference (e.g., `alias: ${cluster-name}`).

Reviewer 2: ```suggestion
* *YAML Variable Parsing:* Fixed a bug in YAML parsing when a value is a variable reference (for example, `alias: ${cluster-name}`).
```
